graphql_api/types/plan/plan.graphql

```diff
@@ -1,20 +1,26 @@
 type Plan {
-  trialStatus: TrialStatus!
-  trialStartDate: DateTime
-  trialEndDate: DateTime
-  trialTotalDays: Int
-  pretrialUsersCount: Int
+  baseUnitPrice: Int!
+  benefits: [String!]!
+  billingRate: PlanBillingRate!
+  hasSeatsLeft: Boolean!
+  isEnterprisePlan: Boolean!
+  isFreePlan: Boolean!
+  isProPlan: Boolean!
+  isSentryPlan: Boolean!
+  isTeamPlan: Boolean!
+  isTrialPlan: Boolean!
```

Comment:
How are all these boolean fields different from using a enum for plan name?

Reply:
A couple aspects to it; first being there are multiple pro / sentry / team / enterprise / free plans so we can't rely on just the name of the plan to uniquely identify the plan type

Secondly, we _could_ use the 'tier name' attribute, and I was thinking about that route as well, but then in the gazebo code instead of having something like 

**BEFORE**
`isFreePlan(accountDetails?.plan?.value)`

**AFTER**
accountDetails?.plan?.isFreePlan

**POTENTIAL AFTER**
accountDetails?.plan?.tier == 'free'

which then kind of leaves us in a similar situation as currently where we have some business logic in the FE. I could go either way tbh, I was just slightly leaning toward keeping gazebo more presentational

Reply:
Ok got it
> there are multiple pro / sentry / team / enterprise / free plans
does this mean multiple of these `is_*` booleans can be true for a given org?